journals/apps/journals/models.py

```diff
@@ -246,6 +248,32 @@ def data(self):
         return contents
 
 
+class JournalImage(AbstractImage):
+    '''
+    Override the base Image model so we can index the Image contents for search
+    and add additional fields
+    '''
+    caption = models.CharField(max_length=255, blank=True)
+
+    search_fields = AbstractImage.search_fields + [
+        index.SearchField('caption', partial_match=True),
+        index.FilterField('id'),
+    ]
+
+    admin_form_fields = Image.admin_form_fields + (
+        'caption',
+    )
+
+
+class JournalImageRendition(AbstractRendition):
```

Comment:
Seems you have to override it when you override AbstractImage, even if you are not changing it. Otherwise you get a db error where it's looking for Image instead of our subclass.
http://docs.wagtail.io/en/v1.13.2/advanced_topics/images/custom_image_model.html

Reply:
Got it. Good to know.